src/docfx/config/Config.cs

```diff
@@ -15,6 +15,11 @@ internal class Config
         private static readonly string[] s_defaultContentInclude = new[] { "docs/**/*.{md,yml,json}" };
         private static readonly string[] s_defaultContentExclude = Array.Empty<string>();
 
+        /// <summary>
+        /// Depot name. = _(ops `DepotName` = `<ProductName>.<DocsetName>`)_
+        /// </summary>
+        public readonly string Name = string.Empty;
```

Comment:
>string.Empty [](start = 38, length = 12)

why is this string.Empty?

Reply:
it is a default value and will be assigned with the user config.

---
In reply to: [194420000](https://github.com/dotnet/docfx/pull/2840#discussion_r194420000) [](ancestors = 194420000)